app/editor/src/css/_colors.scss

```diff
@@ -26,13 +26,23 @@ $border-color: #efefef;
 $action-icon-color: #00478f;
 $darker-background-color: #faf9f7;
 
-
 // Table
 $table-color: #dee2e6;
 $table-header-color: #e9ecef;
 $table-even-row-color: #e9ecef;
 $table-odd-row-color: white;
 
+// Subscriber defaults
+$subscriber-background-color: #efebdf;
```

Comment:
Not sure it makes sense to have subscriber values in the editor.  Might make more sense to have generic names.  
The story to do this was to clean things up.

Reply:
Updated again